lib/src/algorithms/fernet.dart

```diff
@@ -0,0 +1,101 @@
+part of encrypt;
+
+/// Wraps the Fernet Algorithm.
+class Fernet implements Algorithm {
+  final _maxClockSkew = 60;
+  final _randomGenerator = Random.secure();
+  Key _signKey;
+  Key _encryptionKey;
+  Clock _clock;
+
+  Fernet(Key b64key, {Clock clock}) {
+    final keyString = convert.utf8.decode(b64key.bytes);
```

Comment:
I'm in doubt about this lines. Why all this conversion? Just `Key.bytes` isn't enough?

Reply:
same reason as comment https://github.com/leocavalcante/encrypt/pull/66#discussion_r313688914.

`b64key.bytes` is a utf8 encoded, b64 encoded bytes, that's why I must first utf8 decode, and then b64 decode to get back the original `my32lengthsupersecretnooneknows1` key.